src/redisai.c

```diff
@@ -1332,6 +1398,55 @@ int RedisAI_ScriptSet_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv
   return REDISMODULE_OK;
 }
 
+int RedisAI_Info_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+  RedisModule_AutoMemory(ctx);
+
+  if (argc > 1) return RedisModule_WrongArity(ctx);
+
+  long long rstats_size = AI_dictSize(run_stats);
+  RedisModule_ReplyWithArray(ctx, rstats_size);
+
+  AI_dictIterator* iter = AI_dictGetIterator(run_stats);
+  AI_dictEntry* stats_entry = AI_dictNext(iter);
+
+  while (stats_entry) {
+    struct RedisAI_RunStats *rstats = AI_dictGetVal(stats_entry);
+
+    RedisModule_ReplyWithArray(ctx, 14);
+
+    RedisModule_ReplyWithSimpleString(ctx, "KEY");
+    RedisModule_ReplyWithString(ctx, rstats->key);
+    RedisModule_ReplyWithSimpleString(ctx, "TYPE");
+    if (rstats->type == 0) {
+      RedisModule_ReplyWithSimpleString(ctx, "MODEL");
+    }
+    else {
+      RedisModule_ReplyWithSimpleString(ctx, "SCRIPT");
+    }
+    RedisModule_ReplyWithSimpleString(ctx, "BACKEND");
+    RedisModule_ReplyWithSimpleString(ctx, RAI_BackendName(rstats->backend));
+    RedisModule_ReplyWithSimpleString(ctx, "DEVICE");
+    RedisModule_ReplyWithSimpleString(ctx, rstats->devicestr);
+    RedisModule_ReplyWithSimpleString(ctx, "DURATION");
+    RedisModule_ReplyWithLongLong(ctx, rstats->duration_us);
+    RedisModule_ReplyWithSimpleString(ctx, "SAMPLES");
+    if (rstats->type == 0) {
+      RedisModule_ReplyWithLongLong(ctx, rstats->samples);
+    }
+    else {
+      RedisModule_ReplyWithLongLong(ctx, -1);
+    }
+    RedisModule_ReplyWithSimpleString(ctx, "CALLS");
+    RedisModule_ReplyWithLongLong(ctx, rstats->calls);
+
```

Comment:
missing calls that resulted in error. see the above comment. 
IMHO also missing the overall model running latency histogram which is something easy to implement.

Reply:
Thank you @filipecosta90, do you have a reference for an efficient implementation?

Reply:
Like: https://github.com/HdrHistogram/HdrHistogram_c ?

Reply:
hi there @lantiga, yes. Just tested locally the time on adding a value to the histogram and it should be around ~5ns - so we can be very confident that this histogram is extremely efficient. See:
https://github.com/HdrHistogram/HdrHistogram_c/pull/76#issue-366594040